[2025-10-19T07:14:32.406+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-19T07:14:32.447+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-19T07:14:29.960739+00:00 [queued]>
[2025-10-19T07:14:32.464+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-19T07:14:29.960739+00:00 [queued]>
[2025-10-19T07:14:32.466+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-19T07:14:32.494+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_weather_data> on 2025-10-19 07:14:29.960739+00:00
[2025-10-19T07:14:32.512+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3583) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-19T07:14:32.516+0000] {standard_task_runner.py:63} INFO - Started process 3586 to run task
[2025-10-19T07:14:32.512+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'flood_prediction_pipeline', 'fetch_weather_data', 'manual__2025-10-19T07:14:29.960739+00:00', '--job-id', '263', '--raw', '--subdir', 'DAGS_FOLDER/flood_prediction_dag.py', '--cfg-path', '/tmp/tmpsx0l3t1d']
[2025-10-19T07:14:32.518+0000] {standard_task_runner.py:91} INFO - Job 263: Subtask fetch_weather_data
[2025-10-19T07:14:32.595+0000] {task_command.py:426} INFO - Running <TaskInstance: flood_prediction_pipeline.fetch_weather_data manual__2025-10-19T07:14:29.960739+00:00 [running]> on host fcca755cbcb6
[2025-10-19T07:14:32.729+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flood_prediction_pipeline' AIRFLOW_CTX_TASK_ID='fetch_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-10-19T07:14:29.960739+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-19T07:14:29.960739+00:00'
[2025-10-19T07:14:32.731+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-19T07:14:32.788+0000] {logging_mixin.py:188} INFO - âœ… Weather data fetched: {'temp': 23.10189327735715, 'humidity': 96.01803644708671, 'pressure': 1038.7171232702833, 'wind_speed': 13.716064641546877, 'rain_1h': 16.50974929948441, 'rain_3h': 10.06846957897074, 'rain_6h': 72.13608304556811, 'rain_24h': 126.22613250869844, 'rain_intensity': 8.670259141902807, 'temp_delta': -9.852990175969566, 'humidity_delta': 24.154030231715907, 'hour': 7, 'minute': 14, 'second': 32, 'day': 19, 'month': 10, 'year': 2025}
[2025-10-19T07:14:32.789+0000] {python.py:237} INFO - Done. Returned value was: {'temp': 23.10189327735715, 'humidity': 96.01803644708671, 'pressure': 1038.7171232702833, 'wind_speed': 13.716064641546877, 'rain_1h': 16.50974929948441, 'rain_3h': 10.06846957897074, 'rain_6h': 72.13608304556811, 'rain_24h': 126.22613250869844, 'rain_intensity': 8.670259141902807, 'temp_delta': -9.852990175969566, 'humidity_delta': 24.154030231715907, 'hour': 7, 'minute': 14, 'second': 32, 'day': 19, 'month': 10, 'year': 2025}
[2025-10-19T07:14:32.790+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-19T07:14:32.820+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=flood_prediction_pipeline, task_id=fetch_weather_data, run_id=manual__2025-10-19T07:14:29.960739+00:00, execution_date=20251019T071429, start_date=20251019T071432, end_date=20251019T071432
[2025-10-19T07:14:32.853+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-19T07:14:32.873+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-19T07:14:32.876+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
